openssl/aes.go

```diff
@@ -338,27 +279,15 @@ type noGCM struct {
 }
 
 func (c *aesCipher) NewGCM(nonceSize, tagSize int) (cipher.AEAD, error) {
-	if !ExecutingTest() || IsStrictFips() {
```

Comment:
`IsStrictFips` should be moved outside this library and implemented as part of its integration with Go (i.e, in the crypto backend).

Reply:
Agreed.